### PR TITLE
added package.json

### PR DESCRIPTION
**Pull request by jmartin84**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Angular-localizr",
```
Comment (JasonPierce) [inline package.json]: Should this be "Angular-Localizer", just for the sake of consistency?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "main": "angular-localizer.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/2Toad/Angular-Localizer"
+  },
+  "keywords": [
+    "angular",
+    "localization"
+  ],
+  "author": "Jason Pierce",
```
Comment (JasonPierce) [inline package.json]: Thanks, but credit should really go to 2Toad

Comment (jmartin84) [inline package.json]: I know the truth of both toads

File: package.json (hunk 1 above)
Comment (jmartin84) [inline package.json]: I thought this name had more pizzaz

**Comment by JasonPierce** [comment]
Thanks for making it fun. You the man!

**Pull request merged**